### PR TITLE
Added OpenNox

### PR DESCRIPTION
**Pull request by Mailaender**
See also https://www.patreon.com/opennox/ and https://nox.mod.io/

### CODE REVIEW

File: games/o.yaml [hunk 1]
```diff
@@ -1605,6 +1605,18 @@
   type: remake
   updated: 2019-05-29
 
+- name: OpenNox
+  lang:
+  - C#
+  license:
```
Comment (cxong) [inline games/o.yaml:1611]:
```suggestion
  framework:
  - Unity
  license:
```

**Review by cxong — COMMENTED** [review]
I'm puzzled by this entry. The repo and websites don't link to each other, the patreon mentions recent releases whereas the repo has no updates for 8 months. Are the two games even related or just happen to share the same name? Are the authors paywalling recent releases and source code for members?

**Comment by cxong** [comment]
From the patreon page:

> ⚙️Open: keep progress open, rewrite and open-source in the future.

So I don't think this is open source. I don't know the status of the gitlab project though.

**Comment by Mailaender** [comment]
Not picking a license definitely makes this not Open Source according to the official definition. Then again, neither did the Sonic/GTA decompilations. I joined the Discord and there both nicknames match and the repository is linked. This is still not a bright star in Open Source game clone heaven. I admit that.

**Comment by Mailaender** [comment]
They just told me that the Gitlab repository is something else. A recreation in Unity or Godot. The other project is not publicly available and won't be publicly released after all proprietary code is replaced.

**Comment by Mailaender** [comment]
There is also https://snapcraft.io/opennox

**Comment by dennwc** [comment]
It's open source now: https://github.com/noxworld-dev/opennox

**Pull request closed without merging**